### ISSUE TITLE
`TryRuntime` not Work on Pangoro

### ISSUE THREAD

**Issue by AurevoirXavier** [darwinia-network/darwinia-common#821]
```
2021-09-15 15:22:22 `pallet_timestamp::UnixTime::now` is called at genesis, invalid value returned: 0
2021-09-15 15:22:22 [#0] 🗳  Finalized election round with compute ElectionCompute::OnChain.
2021-09-15 15:22:22 [0] 💸 new validator set of size 1 has been processed for era 1
2021-09-15 15:22:22 initializing remote client to "http://pangoro-rpc.darwinia.network"
2021-09-15 15:22:23 scraping keypairs from remote node http://pangoro-rpc.darwinia.network @ 0x78770b41dc4948a95bf581064c4270cabf1bd716bf24217e903b9596f8c001bb
2021-09-15 15:22:23 downloading data for all modules.
2021-09-15 15:22:26 extending externalities with 1 manually injected keys
2021-09-15 15:22:26 injecting a total of 8579 keys
Error: Input("failed to execute 'TryRuntime_on_runtime_upgrade' due to MethodNotFound(\"TryRuntime_on_runtime_upgrade\")")
```

**Comment by AurevoirXavier** [comment]
> Cool. works with `dev`. Why `--chain pangolin` can't work, can you elaborate?
> 
> ```rust
> #[cfg(feature = "try-runtime")]
> 		Some(Subcommand::TryRuntime(cmd)) => {
> 			let runner = cli.create_runner(cmd)?;
> 			let chain_spec = &runner.config().chain_spec;
> 
> 			set_default_ss58_version(chain_spec);
> 
> 			if chain_spec.is_pangoro() {
> 				runner.async_run(|config| {
> 					// we don't need any of the components of new_partial, just a runtime, or a task
> 					// manager to do `async_run`.
> 					let registry = config.prometheus_config.as_ref().map(|cfg| &cfg.registry);
> 					let task_manager = TaskManager::new(config.task_executor.clone(), registry)
> 						.map_err(|e| sc_cli::Error::Service(sc_service::Error::Prometheus(e)))?;
> 
> 					Ok((
> 						cmd.run::<pangoro_runtime::Block, PangoroExecutor>(config),
> 						task_manager,
> 					))
> 				})
> 			} else {
> 				runner.async_run(|config| {
> 					// we don't need any of the components of new_partial, just a runtime, or a task
> 					// manager to do `async_run`.
> 					let registry = config.prometheus_config.as_ref().map(|cfg| &cfg.registry);
> 					let task_manager = TaskManager::new(config.task_executor.clone(), registry)
> 						.map_err(|e| sc_cli::Error::Service(sc_service::Error::Prometheus(e)))?;
> 
> 					Ok((
> 						cmd.run::<pangolin_runtime::Block, PangolinExecutor>(config),
> 						task_manager,
> 					))
> 				})
> 			}
> 		}
> ```

Pangolin's genesis doesn't contain `try-runtime`.

**Comment by flipchan** [comment]
@AurevoirXavier was this ever solved? its because try-runtime was not in the underlaying node i assume(?)

**Comment by AurevoirXavier** [comment]
> @AurevoirXavier was this ever solved? its because try-runtime was not in the underlaying node i assume(?)

Yep, it solved by #869.
